pkg/plugins/policies/meshloadbalancingstrategy/plugin/v1alpha1/plugin_test.go

```diff
@@ -1345,13 +1356,20 @@ var _ = Describe("MeshLoadBalancingStrategy", func() {
 			// then
 			Expect(getResourceYaml(generatedResources.ListOf(envoy_resource.ClusterType))).
 				To(matchers.MatchGoldenYAML(filepath.Join("testdata", fmt.Sprintf("%s.gateway_cluster.golden.yaml", given.name))))
+			Expect(getResourceYaml(generatedResources.ListOf(envoy_resource.EndpointType))).
+				To(matchers.MatchGoldenYAML(filepath.Join("testdata", fmt.Sprintf("%s.gateway_endpoint.golden.yaml", given.name))))
 			Expect(getResourceYaml(generatedResources.ListOf(envoy_resource.ListenerType))).
 				To(matchers.MatchGoldenYAML(filepath.Join("testdata", fmt.Sprintf("%s.gateway_listener.golden.yaml", given.name))))
 			Expect(getResourceYaml(generatedResources.ListOf(envoy_resource.RouteType))).
 				To(matchers.MatchGoldenYAML(filepath.Join("testdata", fmt.Sprintf("%s.gateway_route.golden.yaml", given.name))))
 		},
 		Entry("basic outbound cluster", gatewayTestCase{
 			name: "basic",
+			endpointMap: map[core_xds.ServiceName][]core_xds.Endpoint{
+				"backend": {
+					createEndpointWith("test-zone", "192.168.1.1", map[string]string{}),
```

Comment:
Can we add an endpoint in the local zone and in a different zone to check if it works correctly for `disabled`?

Reply:
added test